models/database.go

```diff
@@ -584,6 +584,9 @@ var databaseSchema = [][]string{
 		FROM services
         WHERE service_type = 'mysql';`,
 	},
+	45: {
+		`ALTER TABLE artifacts ADD COLUMN db_version VARCHAR NOT NULL`,
```

Comment:
this won't work, you need to provide default value, or it will fail on existing records.

Reply:
yup, thanks, default added

Reply:
But also below is said that we shouldn't use default. So I was also dropping default in one migration. But not sure TBH